recipes/ms-gsl/all/conanfile.py

```diff
@@ -37,37 +39,44 @@ def _contract_map(self):
     def _compilers_minimum_version(self):
         return {
             "gcc": "5",
-            "Visual Studio": "14",
             "clang": "3.4",
             "apple-clang": "3.4",
         }
 
+    def validate(self):
```

Comment:
validate() after package_id() please, to understand the flow (package id can remove things in self.info, and it must be taken into account validate()).

Reply:
Done.